src/parser/core/modules/Gauge/TimerGauge.ts

```diff
@@ -250,4 +261,54 @@ export class TimerGauge extends AbstractGauge {
 	setRemoveTimestampHook(value: Analyser['removeTimestampHook']) {
 		this._removeTimestampHook = value
 	}
+
+	private internalDowntime(start = this.parser.pull.timestamp, end = this.parser.currentEpochTimestamp) {
```

Comment:
I know that "downtime" makes sense when talking about time a timer based job gauge isn't running, but the term is also overloaded with unable to act time, and we're going to need to be referring to both when computing how long the job gauge wasn't running for that it should have been, so is there an alternate term we could use here to describe "time the gauge wasn't running or paused"?